src/test/java/com/epam/util/ModelUtilityClass.java

```diff
@@ -0,0 +1,40 @@
+package com.epam.util;
+
+import com.epam.model.*;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+
+public class ModelUtilityClass {
```

Comment:
add a modifier abstract and private constructor

Reply:
Done.